libbeat/beat/errors.go

```diff
@@ -1,30 +0,0 @@
-package beat
-
-import "fmt"
-
-var (
-	// GracefulExit is an error that signals to exit with a code of 0.
-	GracefulExit = ExitError{}
-)
-
-// ExitError is an error type that can be returned to set a specific exit code.
-type ExitError struct {
```

Comment:
Perhaps in an other PR we could use handleError to convert the error to a ExitError and then in the main file it would be:

```
func main() {
	if err := beat.Run(Name, "", beater.New()); err != nil {
		os.Exit(err.ExitCode)
	}
}
```